JoshuaKodhe/Store-Manager
# #161048229 Create UI view to enable admin to create sale attendant

#### What does this PR do?

Allow an admin to add a new store attendant

#### Description of Task to be completed?

Create a section in the profile page that only appears
if a user is an admin
Create a form that allows admin to add a store
attendant

#### How should this be manually tested?

Navigate to the profile.html file to see the user admin 
section
Navigate to the new-attendant-form.html page

#### What are the relevant pivotal tracker stories?

#161048229

#### Screenshots
Admin section in profile.html page
![screenshot from 2018-10-12 11-24-25](https://user-images.githubusercontent.com/43740193/46857535-d3eb9e00-ce11-11e8-8822-80d3198dc8c5.png)

new-attendant-form.html
![screenshot from 2018-10-12 11-28-46](https://user-images.githubusercontent.com/43740193/46857684-2331ce80-ce12-11e8-8410-f2694968991c.png)
